buildenv/docker/mkdocker.sh

```diff
@@ -582,30 +597,46 @@ install_freemarker() {
 }
 
 bootjdk_dirs() {
+  local version
   for version in $@ ; do
     echo /home/$user/bootjdks/jdk$version
   done
 }
 
 bootjdk_url() {
   local version=$1
-  if [ $arch = x86_64 ] ; then
-    echo https://api.adoptopenjdk.net/v3/binary/latest/$version/ga/linux/x64/jdk/openj9/normal/adoptopenjdk
+  if [ $version -lt 17 ] ; then
+    echo https://api.adoptopenjdk.net/v3/binary/latest/$version/ga/linux/${arch/x86_64/x64}/jdk/openj9/normal/adoptopenjdk
   else
-    echo https://api.adoptopenjdk.net/v3/binary/latest/$version/ga/linux/$arch/jdk/openj9/normal/adoptopenjdk
+    echo https://api.adoptium.net/v3/binary/latest/$version/ga/linux/${arch/x86_64/x64}/jdk/hotspot/normal/eclipse
```

Comment:
Kind of a nitpick, but considering `${arch/x86_64/x64}` is used twice, maybe it should just be commoned out into a local var prior to the if/else.

Reply:
Fixed in f5889f7e1c337a7ff88cd70d2a310e43082790a3.